docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_4.rst

```diff
@@ -17,6 +17,11 @@ Release notes: Open GEE 5.3.4
       tasks has been modified to use less memory. This change is experimental
       and can be enabled using the :ref:`GraphOperations flag <graph-operations>`.
 
+      **Option to retry failed tasks**.
+      Users can now configure Fusion to
+      :ref:`retry tasks that fail <retry-failed-tasks>`, which allows
+      Fusion to deal with intermittent failures gracefully.

```

Comment:
Very minor: "deal with intermittent failures gracefully" almost sounds like "fail gracefully" to me. I think something that "which allows Fusion to recover from intermittent failures" may give a more clear description.
But if this is the only comment on the PR, I wouldn't mess with it.

Reply:
I should note clearly, other than this, everything looks great. And again, I wouldn't hold up the PR over this. So I still approve it.
Reviewed only, not tested.